File: iniciante/c99/1011/1011.java
//QUESTÃO 1011 - Esfera

import java.io.IOException;
import java.util.Scanner;
 
public class Main {
	public static void main(String[] args) throws IOException {
		     
        	Scanner sc = new Scanner(System.in);
			            
	        double PI = 3.14159;
	        double raio = sc.nextDouble();
					            
 	        double volume = (4.0/3) * PI * Math.pow(raio, 3);
						            
	        String saida = String.format("VOLUME = %.3f", volume);
							        
	        System.out.println(saida);
	}
}

File: iniciante/c99/1011/1011.c
#include <stdio.h>
#include <math.h>

int main() {

	int raio;
	const double PI = 3.14159;

	scanf("%d", &raio);

	double volume = ((4.0/3) * PI) * (pow(raio, 3));

	printf("VOLUME = %.3lf\n", volume);

	return 0;

}
